pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/plugin/PythonPlugin.groovy

```diff
@@ -189,7 +164,11 @@ class PythonPlugin implements Plugin<Project> {
 
             doLast {
                 def activateLinkSource = VirtualEnvExecutableHelper.getExecutable(settings, "bin/activate")
-                FileSystemUtils.makeSymLink(project, activateLinkSource, settings.getDetails().activateLink)
+                def activateLink = settings.getDetails().activateLink
+                if (activateLink.exists()) {
+                    activateLink.delete()
```

Comment:
What if user uses the same name by mistake?
We should not delete files like this and should fail instead gracefully so that user learns to avoid using the same name we use for this symbolic link.